<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorksheetDock</class>
 <widget class="QWidget" name="WorksheetDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>583</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="2">
        <widget class="KComboBox" name="cbPage"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lHeight">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lGeometry">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Geometry</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lPage">
         <property name="text">
          <string>Page:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="KComboBox" name="cbOrientation"/>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lOrientation">
         <property name="text">
          <string>Orientation:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="TimedLineEdit" name="leName"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lScaleContent">
         <property name="text">
          <string>Scale content:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QCheckBox" name="chScaleContent">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lSizeType">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="cbSizeType"/>
       </item>
       <item row="7" column="2">
        <widget class="NumberSpinBox" name="sbWidth">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="NumberSpinBox" name="sbHeight">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBackground">
      <attribute name="title">
       <string>Background</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>238</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLayout">
      <attribute name="title">
       <string>Layout</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lLayout">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lHorizontalSpacing">
         <property name="text">
          <string>Horizontal:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lBottomMargin">
         <property name="text">
          <string>Bottom:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lMargin">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Margins</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lRightMargin">
         <property name="text">
          <string>Right:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QSpinBox" name="sbLayoutColumnCount">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cbLayout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="13" column="3">
        <widget class="QSpinBox" name="sbLayoutRowCount">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lRowCount">
         <property name="text">
          <string>Rows:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lTopMargin">
         <property name="text">
          <string>Top:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lLeftMargin">
         <property name="text">
          <string>Left:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="lColumnCount">
         <property name="text">
          <string>Columns:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lVerticalSpacing">
         <property name="text">
          <string>Vertical:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lGrid">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Grid</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lSpacing">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Spacing</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>284</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="3">
        <widget class="NumberSpinBox" name="sbLayoutBottomMargin">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="NumberSpinBox" name="sbLayoutTopMargin">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="NumberSpinBox" name="sbLayoutLeftMargin">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="NumberSpinBox" name="sbLayoutRightMargin">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="NumberSpinBox" name="sbLayoutHorizontalSpacing">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="NumberSpinBox" name="sbLayoutVerticalSpacing">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/kdefrontend/widgets/TimedLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>cbSizeType</tabstop>
  <tabstop>cbPage</tabstop>
  <tabstop>cbOrientation</tabstop>
  <tabstop>sbWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>chScaleContent</tabstop>
  <tabstop>cbLayout</tabstop>
  <tabstop>sbLayoutTopMargin</tabstop>
  <tabstop>sbLayoutBottomMargin</tabstop>
  <tabstop>sbLayoutLeftMargin</tabstop>
  <tabstop>sbLayoutRightMargin</tabstop>
  <tabstop>sbLayoutHorizontalSpacing</tabstop>
  <tabstop>sbLayoutVerticalSpacing</tabstop>
  <tabstop>sbLayoutRowCount</tabstop>
  <tabstop>sbLayoutColumnCount</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
